engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/CodePointLengthValidator.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.constraintvalidators.hv;
+
+import org.hibernate.validator.constraints.CodePointLength;
+import org.hibernate.validator.internal.util.logging.Log;
+import org.hibernate.validator.internal.util.logging.LoggerFactory;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+/**
+ * Check that the code point length of character sequence is between min and max.
+ *
+ * @author Kazuki Shimizu
+ */
+public class CodePointLengthValidator implements ConstraintValidator<CodePointLength, CharSequence> {
+
+	private static final Log log = LoggerFactory.make();
```

Comment:
Could you name this `LOG` instead of `log` as it's static (it's probably still wrong in some existing classes but it'd be nice to do it right in new code).